lib/stream.h

```diff
@@ -174,6 +174,7 @@ extern void stream_forward_getp(struct stream *, size_t);
 extern void stream_forward_endp(struct stream *, size_t);
 
 /* steam_put: NULL source zeroes out size_t bytes of stream */
+extern void stream_verify(struct stream *, size_t);
```

Comment:
1. Function comment is now wrong.
2. I don't agree with the purpose of this function. It first truncates the provided size to be the same as the maximum size of the stream, and then asserts if the result isn't <= the maximum writeable size; in other words, it does nothing.